index.js

```diff
@@ -0,0 +1,120 @@
+const model = {
+  todos: [
+    {
+      id: 1,
+      title: "task 1",
+      completed: false,
+      created_at: new Date().toISOString(),
+      updated_at: new Date().toISOString(),
+    },
+    {
+      id: 2,
+      title: "task 2",
+      completed: false,
+      created_at: new Date().toISOString(),
+      updated_at: new Date().toISOString(),
+    },
+    {
+      id: 3,
+      title: "task 3",
+      completed: false,
+      created_at: new Date().toISOString(),
+      updated_at: new Date().toISOString(),
+    },
+  ],
+  nextId: 4, // Next ID to be used
```

Comment:
In order to add a task after deleting all of the tasks, i recommend using this line of code :   model.addTodo({
        id: (model.getTodos()[model.getTodos().length - 1]?.id ?? 0) + 1,
        title: inputElemVal,
        completed: false,
        created_at: "",
        updated_at: "",
      });